### DISCUSSION TITLE
Swift Package Manager support

### DISCUSSION THREAD

**Discussion by richardtop** [nlohmann/json#2806] [Ideas]
[Swift Package Manager](https://swift.org/package-manager/) can be used to integrate together C++ libraries on Apple platforms. For example, with SPM it's possible to [add this project](https://developer.apple.com/documentation/swift_packages/adding_package_dependencies_to_your_app) as a dependency to an iOS app or library.

#### What is the issue you have?

I'd like to use Swift Package Manager to integrate this library into my project.

#### Which compiler and operating system are you using?

- Compiler: Apple Clang / GNU++17 / it doesn't matter
- Operating system: macOS, iOS

#### Which version of the library did you use?

- [x] latest release version 3.9.1
- [ ] other release - please state the version: ___
- [ ] the `develop` branch

**Comment by nlohmann** [comment]
I would be happy to see a PR that integrates that package manager!

**Reply by richardtop** [reply]
The pull request is available here: https://github.com/nlohmann/json/pull/2807
